packages/odyssey-react-mui/src/MenuButton.tsx

```diff
@@ -140,7 +143,7 @@ const MenuButton = ({
   );
 
   return (
-    <div>
+    <div data-se={testId}>
```

Comment:
🤔 maybe this should go on the `Button` instead

Reply:
Yes 👍